app/models/taxon.rb

```diff
@@ -24,14 +26,35 @@ def content_id
     @content_id ||= SecureRandom.uuid
   end
 
+  def path_prefix
+    @path_prefix ||= ''
+  end
+
+  def path_slug
+    @path_slug ||= (title.blank? ? '' : '/' + title.parameterize)
+  end
+
+  def base_path=(base_path)
```

Comment:
This loses the delimiter, which would require putting the getters back in to put the '/' back, or else fiddling around a bit more in this setter. Not convinced this is much cleaner?